percentage-fee-handler/src/lib.rs

```diff
@@ -108,10 +128,18 @@ pub mod pallet {
 		fn get_fee(domain: DomainID, asset: MultiAsset) -> Option<u128> {
 			match (asset.fun, asset.id) {
 				(Fungible(amount), _) => {
-					// return fee rate as 0 when it is not set in the storage
-					let fee_rate_basis_point =
+					// return fee_rate_basis_point as 0 when it is not set in the storage
```

Comment:
I'd suggest returning `None` if no fee has been set for this asset, then the caller can determine how to handle it, likely returning an error

Reply:
Absolutely man, I put a bit thinking here as well but decided to go with default value, letting the caller decide what to do is much better way, will change this. Thanks for the suggestion